public/javascripts/SVLabel/src/SVLabel/Storage.js

```diff
@@ -39,6 +39,10 @@ function TemporaryStorage(JSON, params) {
             set("completedOnboarding", null);
         }
 
+        if (!get("completedMissionAnonymously")){
```

Comment:
We make use of the completedFirstMission function for the init mission instructions.